docker/build.Dockerfile

```diff
@@ -87,12 +87,37 @@ RUN  CXX_COMPILER="g++" \
     && cmake --build . --target check \
     && cmake --build . --target install
 
-# Enable perfomance layers and test with `vkcube`.
+# Enable the perfomance layers and test them with `vkcube`.
 RUN export LD_LIBRARY_PATH=/performance-layers/build:$LD_LIBRARY_PATH \
     && export VK_INSTANCE_LAYERS=VK_LAYER_STADIA_pipeline_compile_time \
     && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_runtime \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_cache_sideload \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_memory_usage \
     && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_frame_time \
+    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_memory_usage \
+    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_cache_sideload \
     && export VK_LAYER_PATH=/performance-layers/layer \
-    && xvfb-run vkcube --c 100 && echo Done!
+    && export VK_COMPILE_TIME_LOG=/performance-layers/build/compile_time.csv \ 
+    && export VK_RUNTIME_LOG=/performance-layers/build/run_time.csv \ 
+    && export VK_FRAME_TIME_LOG=/performance-layers/build/frame_time.csv \
+    && export VK_MEMORY_USAGE_LOG=/performance-layers/build/memory_usage.csv \
+    && export VK_PIPELINE_CACHE_SIDELOAD_FILE=/performance-layers/build/cache_sideload_file.csv \
+    && export VK_PERFORMANCE_LAYERS_EVENT_LOG_FILE=/performance-layers/build/events.log \
+    && touch /performance-layers/build/cache_sideload_file.csv \
+    && xvfb-run vkcube --c 100 && echo Run is completed! 
+
+# Check if each enabled layer is writing in its corresponding log file.
+RUN if [ `wc -l <compile_time.csv` -lt 2 ]; then \
```

Comment:
Since the check appears to be the same for each of the layers, can we make it a function?

Reply:
Also, prefer subshells in the form of `$(your cmd)` over backticks. It comes with fewer surprised and allows for nesting. See http://mywiki.wooledge.org/BashFAQ/082